src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<blockquote>which we will modify to account for the fact that the wing mass is computed on its own. <s>and add said wing</s></blockquote><p>Looks enough to me</p><p> </p><blockquote>where MTOW is in <strong>lbf </strong>and</blockquote><p>I guess it's about pounds (lb), not pounds-force (lbf)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
> Looks enough to me

Corrected in 7d280c2d 

> I guess it's about pounds (lb), not pounds-force (lbf)

In the book its actually lbf. Even so 1lbf = 1 lb, no ?

Reply:
<p><a href="https://en.wikipedia.org/wiki/Pound_(force)#Product_of_avoirdupois_pound_and_standard_gravity" target="_blank">Not at all!</a></p><p> </p><p>Despite the usage of the word "weight", we are used to providing a mass value for MTOW. But using an actual force value, in N or lbf, could make sense. </p><p>But as long it is about mass, it will be kg or lb. </p><p>So how is the formula supposed to be used? Since the formula has no dimensional homogeneity, this is very important to know.</p><p> </p><p> </p>

Reply:
So ...
One author says lbs (implied pound mass), the author says $lb_f$ for the same formula. The results however are very far from statistics when using $lb_f$ so I suggest we go with lbs.

Alternatively we could go for another formula which uses only MTOW, AR and $S_w$, the domain of validity is much smaller though

Reply:
Done in 863446b0